crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -222,23 +222,25 @@ pub enum Error {
 /// ### Console logs
```

Comment:
As discussed on the call, we could add a comment mentioning that the arguments in parentheses are only applicable when the clap feature is enabled.

Reply:
Done in 367580ce57a21c2ca45c334caac35ea25f9711c2.